features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/ecp/ecp_internal_alt.c

```diff
@@ -53,6 +53,7 @@
  *        would be defined in mbedtls/ecp.h from ecp.c for our inclusion */
 #define ECP_SHORTWEIERSTRASS
 
+#include "mbedtls/ssl.h"
```

Comment:
I think it would be better to include just`"mbedtls/platform.h"` here.

Reply:
@yanesca I fix it in https://github.com/ARMmbed/mbed-os/pull/9081/commits/b16b1dbeb972c61d803982d62f136889bc11b1ca.